### DISCUSSION TITLE
Console platforms support

### DISCUSSION THREAD

**Discussion by Skeltpr** [nxrighthere/UnrealCLR#126] [General]
I am extremely interested in developing using UnrealCLR. Fast-paced development using Unreal's feature rich engine with a feature rich language sounds delightful. However, as someone who's completely oblivious when it comes to the lower-level interworking's of projects such as this, I wanted to know the likelihood of console support? 

More specifically, how easily could it be done? Worst case scenario, if a game is developed, would it be easy to hire someone to port/publish UnrealCLR to a console using whatever C# console compiler is available? Or to finish that aspect of the UnrealCLR project?

Or is UnrealCLR completely helpless until certain .NET technologies/features are created and supported? (AOT, etc.)

**Comment by nxrighthere** [comment]
> Or is UnrealCLR completely helpless until certain .NET technologies/features are created and supported? (AOT, etc.)

As long as we don't have official support from Microsoft for this kind of things, we can't do much. Even companies that have the resources to invest in areas like ahead-of-time complication are struggling because .NET is evolving all the time.
